StlToDatTranslator/Header File/Triangle.h

```diff
@@ -9,3 +9,7 @@ class Triangle {
     Triangle(Point vertex1, Point vertex2, Point vertex3);
```

Comment:
as you are keeping data members private in point class you should also do it in this class and provide the getter function for the same

Reply:
Triangle data members should be private.